job-dsls/src/main/resources/job-scripts/prod_rhba_staging.jenkinsfile

```diff
@@ -136,8 +136,10 @@ def createStagingPath(path) {
 }
 
 def stageArtifact(url, path) {
-    println "[INFO] staging ${url} to ${path}"
-    remoteExec("curl -s ${url} -o ${path}")
+    // need to use indy-admin here as there's a caching bug which can make some downloads fail
+    def indyAdmin = url.replace("indy", "indy-admin")
```

Comment:
Just out of curiosity, is "indy" always expected as part of the url? if it is not in the url is the replace going to fail and stop the execution?

Reply:
yes, indy is always expected here in the download url